### ISSUE TITLE
change env var DIUN_NOTIF_SLACK_WEBHOOKURL to annotations

### ISSUE THREAD

**Issue by razvan-moj** [crazy-max/diun#322]
if multiple uses in a cluster want to receive update notifications, but to different Slack channels, it would be nice to have a way to specify the recipient per deployment/pod rather than - what we do now - run multiple instances of Diun, which seems wasteful.

**Comment by crazy-max** [comment]
@razvan-moj Yes we could allow notifiers per containers through labels/annotations.

**Comment by kireque** [comment]
This option would be very useful for me! 
At the moment I use MQTT but everything is published in the same topic. Would be great if there was a separate topic for each container.

**Comment by x3ntrix** [comment]
Such an option would be useful to use too.
We have a docker server which hosts containers from different teams and each team is responsible to keep their containers up-to-date. For notifications, we use mails. The complete mail configuration can be the same in our case, but it would be great to specify/override the mail recipient via a label for each container, so a team receives only mails for their containers.
